### ISSUE TITLE
🐛 BUG: Error code 10002 on wrangler publish

### ISSUE THREAD

**Issue by iandees** [cloudflare/wrangler2#2454]
### What version of `Wrangler` are you using?

2.6.2

### What operating system are you using?

macOS 13.1

### Describe the Bug

Running `wrangler publish` produces an internal error:

```
✘ [ERROR] A request to the Cloudflare API (/accounts/abb981ba94fd6c090352b253fe594676/workers/scripts/s3-to-r2) failed.

  workers.api.error.internal_server [code: 10002]

  If you think this is a bug, please open an issue at:
  https://github.com/cloudflare/wrangler2/issues/new/choose
```

**Comment by rozenmd** [comment]
Hey @iandees sorry you've run into this issue - is it still happening for you?

**Comment by iandees** [comment]
Nope, doesn't look like it.

For context, I was trying to deploy [this code](https://github.com/KianNH/cloudflare-worker-s3-to-r2-migration) as a worker. It worked, I made a configuration change to point to my own R2 and S3 buckets, and then I started getting the above error when publishing. Even when I undid my configuration changes it kept failing. I `git reset --hard HEAD` to remove all my changes, and it worked again. Since there wasn't any feedback on what was causing the error, I don't know which of my changes made it fail.
